modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -614,16 +671,34 @@ return view.extend({
 			so.value(ipv4, ipaddrs[ipv4] ? '%s (%s)'.format(ipv4, ipaddrs[ipv4]) : ipv4);
 		});
 
-		so = ss.option(form.Value, 'leasetime', _('Lease time'));
+		so = ss.option(form.Value, 'leasetime', 
+			_('Lease time'),
+			_('Host-specific lease time, e.g. <code>2m</code>, <code>3h</code>, <code>5d</code>.'));
```

Comment:
We could add here the following line to give the user a Drop down with prefilled values.
```
so.value('2m', _('2 minutes'));
so.value('3h', _('3 houres'));
```

Reply:
Done.